inlong-dashboard/src/plugins/sinks/common/SinkDefaultInfo.ts

```diff
@@ -166,17 +185,40 @@ export class SinkDefaultInfo implements DataWithBackend, RenderRow, RenderList {
     return FieldList.filter(item => SyncCreateTableFieldSet.has(item.name as string));
   }
 
-  renderRow() {
+  renderSyncAllRow() {
     const constructor = this.constructor as typeof SinkDefaultInfo;
-    constructor.FieldList.filter(item => {
-      if (item.name === 'tableName' || item.name === 'primaryKey' || item.name === 'database') {
-        item.type = 'input';
+    const { FieldList, SyncMoveDbFieldSet } = constructor;
+    return FieldList.filter(item => {
+      if (item.name === 'sinkType') {
+        item.props = values => ({
+          disabled: Boolean(values.id),
+          dropdownMatchSelectWidth: false,
+          options: sinks
+            .filter(item => item.value === 'ICEBERG')
+            .map(item => ({
+              label: item.label,
+              value: item.value,
+              image: loadImage(item.label),
+            })),
+        });
       }
-      if (item.name === 'createTableField') {
-        item.hidden = true;
+      return SyncMoveDbFieldSet.has(item.name as string);
+    });
+  }
+
+  renderRow() {
+    const constructor = this.constructor as typeof SinkDefaultInfo;
+    return constructor.FieldList.filter(item => {
```

Comment:
1. The result of filter needs to return a boolean value
2. I do not recommend placing the fields of specific instances such as backupTable, backupDatabase, tableName, and primaryKey in abstract methods. This will cause the functions of each plug-in to be gradually coupled together.